### PR TITLE
head's args aren't in bytes

### PR DESCRIPTION
**Pull request by Parth-Mehrotra**

### CODE REVIEW

File: pages/linux/head.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 - Output the first few bytes of a file
 
-`head -c {{size in kilobytes}} {{filename}}`
+`head -c {{size in bytes}} {{filename}}`
```
Comment (igorshubovych) [inline pages/linux/head.md]: Interesting. I would rather put something like
{{bytes_size}}

Comment (waldyrious) [inline pages/linux/head.md]: That wouldn't be as intuitive, IMO. But the token should indeed use underscores as word separators, @Parth-Mehrotra, so I'd suggest `{{size_in_bytes}}`.

**Comment by Parth-Mehrotra** [comment]
Okay @waldyrious @igorshubovych, just replaced all the spaces with underscores

**Comment by igorshubovych** [comment]
Please squash commits.

In future you don't need to squash commits if you do
```shell
git add .
git commit --amend
git push --force
```
This way you are updating your commit and overwriting it in published branch.

**Comment by Parth-Mehrotra** [comment]
Squashed, and updated @igorshubovych

**Comment by felixonmars** [comment]
There are still 2 commits atm, and it should be one.

**Comment by Parth-Mehrotra** [comment]
Good? Or do I need to update this branch as well? @felixonmars

**Comment by waldyrious** [comment]
I now see only one commit, but the diff shows no underscores...

**Comment by Parth-Mehrotra** [comment]
Okay, sorry, never needed to squash commits before and I'm apparently really bad at it haha

**Comment by felixonmars** [comment]
It looks good now, however it shows "This branch is out-of-date with the base branch" and I cannot merge.

**Comment by waldyrious** [comment]
I also see a merge conflict message, but apparently @igorshubovych sees something different in some cases (still a mystery to me how that can happen), so hopefully he can merge.

**Pull request merged**